properties/base.py

```diff
@@ -81,12 +83,20 @@ def __new__(mcs, name, bases, classdict):                                  #pyli
             prop.name = key
             classdict[key] = prop.get_property()
 
-        # Overwrite handlers with their function
+        # Ensure observed names are valid
+        for key, hand in iteritems(observer_dict):
```

Comment:
Hah, @bsmithyman gave me a bit of a hard time about this too. I'll use `handler`. Thanks.